ml_perf/flags/19/rl_loop.flags

```diff
@@ -15,5 +15,8 @@
 
 --eval_num_games=100
 
---min_games_per_iteration=8192
 --validate=1
+
+# Consider also updating num_games in bootstrap.flags if updating
+# min_games_per_iteration.
+--min_games_per_iteration=8192
```

Comment:
why is this duplicated in train_loop?

Reply:
`rl_loop.flags` is deprecated (along with `ml_perf/reference_implementation.py`, `cc/selfplay.cc`, etc). I'm keeping the files around for now as reference but I'll do a clean up pass once the code for MLPerf 0.7 is settled.

Reply:
Sounds good.  As long as multi-vm can fallback it sounds like an elegant solution! :thumbsup: